src_assets/common/assets/web/apps.html

```diff
@@ -1,370 +1,384 @@
-<div id="app" class="container">
-  <div class="my-4">
-    <h1>Applications</h1>
-    <div>Applications are refreshed only when Client is restarted</div>
-  </div>
-  <div class="card p-4">
-    <table class="table">
-      <thead>
-        <tr>
-          <th scope="col">Name</th>
-          <th scope="col">Actions</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr v-for="(app,i) in apps" :key="i">
-          <td>{{app.name}}</td>
-          <td>
-            <button class="btn btn-primary" @click="editApp(i)">
-              <i class="fas fa-edit"></i> Edit
-            </button>
-            <button class="btn btn-danger" @click="showDeleteForm(i)">
-              <i class="fas fa-trash"></i> Delete
-            </button>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
-  <div class="edit-form card mt-2" v-if="showEditForm">
-    <div class="p-4">
-      <!--name-->
-      <div class="mb-3">
-        <label for="appName" class="form-label">Application Name</label>
-        <input
-          type="text"
-          class="form-control"
-          id="appName"
-          aria-describedby="appNameHelp"
-          v-model="editForm.name"
-        />
-        <div id="appNameHelp" class="form-text">
-          Application Name, as shown on Moonlight
-        </div>
-      </div>
-      <!--output-->
-      <div class="mb-3">
-        <label for="appOutput" class="form-label">Output</label>
-        <input
-          type="text"
-          class="form-control monospace"
-          id="appOutput"
-          aria-describedby="appOutputHelp"
-          v-model="editForm.output"
-        />
-        <div id="appOutputHelp" class="form-text">
-          The file where the output of the command is stored, if it is not
-          specified, the output is ignored
-        </div>
-      </div>
-      <!--prep-cmd-->
-      <div class="mb-3">
-        <label for="excludeGlobalPrep" class="form-label"
-          >Global Prep Commands</label
-        >
-        <select
-          id="excludeGlobalPrep"
-          class="form-select"
-          v-model="editForm['exclude-global-prep-cmd']"
-        >
-          <option v-for="val in [false, true]" :value="val">
-            {{ !val ? 'Enabled' : 'Disabled' }}
-          </option>
-        </select>
-        <div class="form-text">
-          Enable/Disable the execution of Global Prep Commands for this
-          application.
-        </div>
-      </div>
-      <div class="mb-3">
-        <label for="appName" class="form-label">Command Preparations</label>
-        <div class="form-text">
-          A list of commands to be run before/after this application.<br />
-          If any of the prep-commands fail, starting the application is aborted.
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <%- header %>
+    <%- headerMain %>
+      <style>
+        .precmd-head {
```

Comment:
Can we put all the styles (throughout) into a custom css file?

Reply:
The custom styles are specific to each page at the moment. I can setup the common css system but Having all the styles, markup and scripts for the specific page on the same file is very similar to the Single File Component approach that Vue uses.